crates/chronicle-protocol/src/protos/submission.proto

```diff
@@ -18,7 +18,18 @@ message OpaPolicy {
 message Submission {
   string version = 1;
   uint64 span_id = 2;
-  repeated string body_old = 3 [deprecated = true];
-  string identity = 4;
-  string body = 5;
+  repeated string body_old = 3 [ deprecated = true ];
+  oneof identity_variant {
+    string identity_old = 4 [ deprecated = true ];
+    IdentityMessageV1 identity = 5;
+  }
+  BodyMessageV1 body = 6;
+}
+
+message BodyMessageV1 {
+  string json = 1;
```

Comment:
Don't name fields after types.  Particularly in this case.  just use `body`, or even better `payload`